docs/platform_introduction/studies.md

```diff
@@ -1,43 +1,55 @@
 
-![Studies](https://user-images.githubusercontent.com/28002886/137332242-079e844a-75b5-4bbb-8f8e-aff89769669c.png)
+![Studies](https://user-images.githubusercontent.com/28002886/153439343-536f50db-c0bc-4bf7-b7a8-4ce559280683.png)
 
 ## Operations from the Dashboard
 
 There are a number of operations that are specific to Studies, accessible by the three dot button at the upper right hand corner of each Study.
 
-![studyoptions](https://user-images.githubusercontent.com/28002886/137308800-0b7b4c5d-c04d-493b-a073-dc60c228b445.png ':size=200')
+![studyoptions](https://user-images.githubusercontent.com/28002886/153577667-fa700a2f-cb28-4d33-bd0d-485d66b3897d.png ':size=200')
+
+<!-- ![studyoptions](https://user-images.githubusercontent.com/28002886/153412169-d8749232-84f6-4db4-a80f-d9aceab47ba7.png ':size=200') -->
 
 These options are:
-1. **Select:** allows you to select this Study and/or others, primarily useful for deletion
-2. **More Info:** allows you to view metadata for a Study including its title, description and icon - editing may also be possible if you have appropriate permissions
-3. **Sharing** allows you view who the Study is shared with and to add collaborators to the Study. See [Sharing Studies](/docs/study_setup/sharestudy.md) for more details.
-4. **Quality:** allows you to view (and edit depending on your permissions) the quality of a Study according to the [Ten Simple Rules](https://www.imagwiki.nibib.nih.gov/content/10-simple-rules-conformance-rubric) for computational modeling.
-5. **Classifiers** allows you to assign classifiers (curated descriptors) for a Study which can then be used for search and filtering using the filter tool on the [Dashboard](docs/platform_introduction/dashboard.md). See [Annotations](/docs/platform_introduction/user_setup/annotations.md) for more details. 
-6. **Services:** allows you to view the Services and their versions that are a part of this Study. You will also be able to see if there are newer versions of these Services available in the Service Catalog.
-7. **Duplicate** allows you to create a copy of that Study which will be renamed with a timestamp.
-8. **Export** allows you to download the Study as a *.osparc* file which can then be imported back onto the platform by yourself or anyone who has that file
-9. **Publish as Template** if selected, you will be allowed to create a copy of the Study in its current state which will appear as a Template Study in the [Discover](docs/platform_introduction/Discover.md) tab for users you share the Template with.* 
-10. **Delete** as you may guess, this will delete the particular Study, with a confirmation popup. 
+1. **Select:** allows you to select this Study and/or others, primarily useful for deletion.
+2. **Rename:** allows you to rename the study
+<!-- 3. **Duplicate** allows you to create a copy of that Study which will be renamed with a timestamp. -->
```

Comment:
We will activate those soon.

TIP: in markdown, if you leave all the enumerations as  ``1. `` you do not need to adjust them anymore when you add/remove items